cloud/aws/metrics/metrics_scraper.Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM public.ecr.aws/aws-observability/aws-otel-collector:latest
+COPY adot-config.yaml /etc/ecs/otel-config.yaml
```

Comment:
I have little context here, so apologies if the answer is obvious.

Can we rename `adot-config.yaml` to `otel-config.yaml` in the repository? Also, it may be useful to have a comment explaining what `adot` / `otel` mean and a link to the docs for valid configuration values.

Reply:
done